src/moveit_visual_tools.cpp

```diff
@@ -96,48 +103,51 @@ bool MoveItVisualTools::loadPlanningSceneMonitor()
   // Check if we already have one
   if (psm_)
   {
-    ROS_WARN_STREAM_NAMED(LOGNAME, "Will not load a new planning scene monitor when one has "
+    RCLCPP_WARN_STREAM(LOGGER, "Will not load a new planning scene monitor when one has "
                                    "already been set for Visual Tools");
     return false;
   }
-  ROS_DEBUG_STREAM_NAMED(LOGNAME, "Loading planning scene monitor");
+  RCLCPP_INFO_STREAM(LOGGER, "Loading planning scene monitor");
 
   // Create tf transform buffer and listener
-  std::shared_ptr<tf2_ros::Buffer> tf_buffer = std::make_shared<tf2_ros::Buffer>();
-  std::shared_ptr<tf2_ros::TransformListener> tf_listener = std::make_shared<tf2_ros::TransformListener>(*tf_buffer);
+  // std::shared_ptr<tf2_ros::Buffer> tf_buffer = std::make_shared<tf2_ros::Buffer>();
+  // std::shared_ptr<tf2_ros::TransformListener> tf_listener = std::make_shared<tf2_ros::TransformListener>(*tf_buffer);
+
+  std::shared_ptr<tf2_ros::Buffer> tf_buffer =
+      std::make_shared<tf2_ros::Buffer>(node_->get_clock(), tf2::durationFromSec(10.0));
+  std::shared_ptr<tf2_ros::TransformListener> tfl = std::make_shared<tf2_ros::TransformListener>(*tf_buffer);
 
   // Regular version b/c the other one causes problems with recognizing end effectors
```

Comment:
... I wonder what this was for

Reply:
I don't have much idea either on that one.